AndroidSDKCore/src/main/java/com/leanplum/internal/LeanplumInternal.java

```diff
@@ -673,6 +674,14 @@ public static boolean getIsScreenTrackingEnabled() {
     return isScreenTrackingEnabled;
   }
 
+  public static boolean getIsVariantDebugInfoEnabled() {
+    return isVariantDebugInfoEnabled;
+  }
+
+  public static void setIsVariantDebugInfoEnabled(boolean _isVariantDebugInfoEnabled) {
+    isVariantDebugInfoEnabled= _isVariantDebugInfoEnabled;
```

Comment:
`this. isVariantDebugInfoEnabled= isVariantDebugInfoEnabled;`

No need to prefix with `_`

Reply:
^

Reply:
static var :(